module/zcommon/zfs_prop.c

```diff
@@ -350,6 +350,9 @@ zfs_prop_init(void)
 	zprop_register_index(ZFS_PROP_CANMOUNT, "canmount", ZFS_CANMOUNT_ON,
 	    PROP_DEFAULT, ZFS_TYPE_FILESYSTEM, "on | off | noauto",
 	    "CANMOUNT", canmount_table);
+	zprop_register_index(ZFS_PROP_QUORUM, "quorum", 0, PROP_INHERIT,
```

Comment:
enum should be good to set the quorum like ZFS_QUORUM_OFF, ZFS_QUORUM_ON

Reply:
for boolean value(on/off) we don't need to have separate enum. ZFS uses boolean_table for that purpose.